.github/workflows/_nodejs.condo.core.tests.yml

```diff
@@ -27,31 +27,63 @@ jobs:
           username: ${{ secrets.SBERCLOUD_CR_USERNAME }}
           password: ${{ secrets.SBERCLOUD_CR_PASSWORD }}
 
-      - name: run postgresql for tests
+      - name: run postgresql master
         run: |
-          docker run -e POSTGRES_USER=$POSTGRES_USER -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_DB=$POSTGRES_DB -p="127.0.0.1:5432:5432" -d swr.ru-moscow-1.hc.sbercloud.ru/doma/utils/postgres:13.2
+          touch ./pg_hba.conf
```

Comment:
This script looks very complex. Ideally, I'd like to see something like `docker-compose up postgres postgres-replica`, and all the magic with replication should happen automatically. This way, there's no need to do it all manually in CI, and it can be properly tested locally.

Reply:
Yup, that's true. But for now we can't run compose modules with self-hosted runner. This will be fixed in the future updates

Reply:
This problem will be solved here -> https://github.com/open-condo-software/condo/pull/4883